src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -33,28 +33,22 @@
  * Handles communication with the keyserver.
  */
 public class Rscc {
+  public static final int PACKAGE_SIZE = 10000; // needed, since a static method access it.
```

Comment:
@l-marchesi  I tested  the following successfully:
Make it private:
  private static final int PACKAGE_SIZE = 10000; 

Make a static getter Method:
  public static int getUdpPackageSizeStatic() {
    return PACKAGE_SIZE;
  }

and access this method in the other class:
  Rscc.getUdpPackageSizeStatic()

This is working, bbut you also can go with the "public final". thats just cosmetics in the end.